style.css

```diff
@@ -61,10 +66,56 @@ body {
   background-color: #ff6b00;
 }
 
+.headline {
+  background-color: transparent;
+  max-width: 60vh;
+  height: 60vh;
+  position: relative;
+  left: 10%;
+  top: 20vh;
+  right: 20%;
+}
+
+.social-media {
+  display: flex;
+}
+
+ul {
+  list-style: none;
+  padding: 0;
+}
+
+.social-media a {
+  cursor: pointer;
+}
+
 @media (min-width: 375px) {
```

Comment:
[OPTIONAL] Since we are following the mobile-first approach, there is no need to use mobile media query. Otherwise, your code will not DRY and you will have to repeat CSS values for mobile and desktop versions.